sympl/_core/constants.py

```diff
@@ -119,5 +240,21 @@ def get_constant(name, units):
     return constants[get_alias(name)].to_units(units).values.item()
 
 
+def get_constants_string():
```

Comment:
Why not just have a function that prints the constants? `sp.print_constants()` seems more usable rather than `print(sp.get_constants_string())`. I'm suggesting this in addition to `sp.get_constants_string()`.

Reply:
I think print(sp.get_constants_string()) is easy enough, and prevents the user from having to remember two function names instead of one. There are other ways a user might want to log the constants string than printing, so it's necessary to have a function like this.